lib/linguist/languages.yml

```diff
@@ -3289,6 +3289,21 @@ Jupyter Notebook:
   aliases:
   - IPython Notebook
   language_id: 185
+Justfile:
+  type: programming
+  aliases:
+  - Justfile
+  color: "#384d54"
+  tm_scope: source.justfile
+  extensions:
+  - ".justfile"
```

Comment:
There are only 34 files with this extension on GitHub. This is no where near popular enough. An extension isn't needed so you can remove this section as the Justfile filename is popular enough.

```suggestion
```

Reply:
That's the search result for the filename, not the extension. [This search](https://github.com/search?q=extension%3Ajustfile&type=Code) reveals only 34 files using the `.justfile` extension.

It doesn't matter what the language accepts... it's the actual usage that matters.

Reply:
Ok, support for `Justfile` and `justfile` has been added.

Reply:
You didn't remove the extension as requested. It's still not popular enough.

Reply:
The file has no extension, it's just "Justfile". In 1 month since this PR was opened, the usage for that went from `5k` to `6k`.
[This](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#adding-a-language) says:

> We try only to add languages once they have some usage on GitHub. In most cases we prefer that each new file extension be in use in at least 200 unique :user/:repo repositories before supporting them in Linguist (but see https://github.com/github/linguist/issues/5756 for a temporary change in the criteria).

@casey, can you please also give us your thoughts to my previous posts in this thread?

Reply:
> The file has no extension,

Exactly! Which means we don't need the `extension` section. Please remove it.

As I've said before, you are mixing up the extention with the filename. `foo.justfile` is _not_ the same as `justfile`. The former does not have enough usage. The latter does.

Reply:
Done.
Do you still feel it's not popular enough, though? :confused: